spec/feature/unauthenticated_guest_spec.rb

```diff
@@ -3,16 +3,26 @@
 describe "An unauthenticated user" do
   include Capybara::DSL
 
-  let(:category1) { Category.create(name: "Breakfast") }
-  let(:category2) { Category.create(name: "Lunch") }
-
-  before(:each) do
+  let!(:category1) { Category.create(name: "Breakfast") }
+  let!(:category2) { Category.create(name: "Lunch") }
+  let!(:item) do
     category1.items.create(title: "Bacon and Eggs",
                            description: "The classic breakfast dish",
                            price: 1000, image: "bacon_and_eggs.jpg")
+  end
+  let!(:item2) do
     category2.items.create(title: "BLT",
                            description: "The classic lunch dish",
                            price: 1000, image: "blt.jpg")
+  end
+  let!(:valid_user) do
+    User.create(first_name: "Alice",
+                last_name: "Smith",
+                email: "rich@gmail.com",
+                password: "password")
+  end
+
+  before(:each) do
```

Comment:
Need to accommodate modification to test below.

_Chaos reigns within. 
Reflect, repent, and reboot. 
Order shall return. _